src/index.html

```diff
@@ -5,8 +5,347 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap" rel="stylesheet">
+
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="header__logo">
+        <img src="./images/logo.png" alt="logo Moyo">
+      </a>
+      <nav class="nav">
```

Comment:
Don't forget to add empty lines between multiline sibling blocks of HTML. Check it everywhere
```suggestion
      </a>
      
      <nav class="nav">
```